rfcs/0003-timestamps-and-ttl.md

```diff
@@ -264,32 +251,79 @@ pub struct KeyValueMeta {
 
 ### Compaction
 
+#### TTL Compaction Filter
+
 We will introduce the concept of a compaction filter, which serves as a mechanism for pluggable
 participation in the compaction process. Specifically, the filter (if configured) will run on each
 key during the compaction process and optionally remove or modify the value.
 
 For the scope of this RFC, we will be introducing a builtin TS/TTL compaction filter. This filter
-will add a tombstone to any value where `db.clock.now() > kv.meta.ts + kv.meta.ttl`.
+will add a tombstone to any value where `db.clock.now() > kv.meta.ts + kv.meta.ttl`. Note that
+tombstones will still contain a timestamp see [out of order insertions](#insertion-timestamps).
```

Comment:
Should disregard TTLs on tombstones, too.

Reply:
Just to make sure we're on the same page here, you mean the compaction filter should remove the TTL from the metadata if it converts it to a tombstone? That was my intention anyway 😅